src/ServiceConfig.py

```diff
@@ -135,14 +135,14 @@ def create_configuration(self):
 
     @staticmethod
     def get_service_port():
-        port = None
-        if os.path.exists("docker-compose.yml"):
-            with open("docker-compose.yml", "r") as f:
-                docker_yml = yaml.safe_load(f)
-                services = list(docker_yml["services"].keys())
-                port = docker_yml["services"][services[0]]["ports"][0].split(":")[0]
-
-        return port
+        # port = None
+        # if os.path.exists("docker-compose.yml"):
+        #     with open("docker-compose.yml", "r") as f:
+        #         docker_yml = yaml.safe_load(f)
+        #         services = list(docker_yml["services"].keys())
+        #         port = docker_yml["services"][services[0]]["ports"][0].split(":")[0]
+
```

Comment:
This commented lines are supposed to be kept?

Reply:
no, removed.